telegram/webhookinfo.py

```diff
@@ -40,13 +40,20 @@ class WebhookInfo(TelegramObject):
 
     """
 
-    def __init__(self, url, has_custom_certificate, pending_update_count, **kwargs):
+    def __init__(self,
+                 url,
+                 has_custom_certificate,
+                 pending_update_count,
+                 last_error_date=None,
+                 last_error_message=None,
```

Comment:
We're missing optionals: `max_connections` (`int`) & `allowed_updates` (`List[str]`)